internal/cli/cli.go

```diff
@@ -67,83 +68,32 @@ type deleteOpts struct {
 	failMessage    string
 }
 
-// DeleterFromProject a function to delete from the store.
-type DeleterFromProject func(projectID string, entry string) error
-
-// DeleterFromProjectAuthDB a function to delete from the store.
-type DeleterFromProjectAuthDB func(authDB string, projectID string, entry string) error
-
-// DeleteFromProject deletes a resource from a project, it expects a callback
+// Delete deletes a resource not associated to a project, it expects a callback
 // that should perform the deletion from the store.
-func (opts *deleteOpts) DeleteFromProject(d DeleterFromProject, projectID string) error {
+func (opts *deleteOpts) Delete(d interface{}, a ...string) error {
 	if !opts.confirm {
-		opts.printFailMessage()
+		fmt.Println(opts.FailMessage())
 		return nil
 	}
-	err := d(projectID, opts.entry)
-
-	if err != nil {
-		return err
-	}
-
-	opts.printSuccessMessage()
-
-	return nil
-}
 
-// DeleterFromProjectAuthDB deletes a resource from a project, it expects a callback
-// that should perform the deletion from the store.
-func (opts *deleteOpts) DeleterFromProjectAuthDB(d DeleterFromProjectAuthDB, authDB, projectID string) error {
-	if !opts.confirm {
-		opts.printFailMessage()
-		return nil
+	var err error
+	fmt.Printf("%#v", d)
```

Comment:
[qa] why do we have a print here?

Reply:
ha, good catch, I think this is a left over from em doing some testing on why typing wouldn't work